packages/overdrive/lib/components/SelectInput/SelectInput.treat.ts

```diff
@@ -0,0 +1,15 @@
+import { style } from 'treat';
+
+export const input = style({});
+export const paddedInput = style(({ space }) => ({
+	selectors: {
+		[`${input}&`]: {
+			paddingRight: space['8'],
+		},
+	},
+}));
```

Comment:
Think this block can simply be: 
```suggestion
export const paddedInput = style(({ space }) => ({
	paddingRight: space['8'],
}));
```